docs/make-docs

```diff
@@ -343,6 +352,7 @@ POSIX_HERESTRING
     --name "${DOCS_CONTAINER}" \
     --platform linux/amd64 \
     --publish "${DOCS_HOST_PORT}:3002" \
+    --publish "3003:3003" \
```

Comment:
where is the "3003" from?

Reply:
Good question,

I promise for future updates there will be a full CHANGELOG to link to now that the procedure is somewhat stable. 

The 3003 port is actually where Hugo listens in the `grafana/docs-base` container image. On 3002 there is an image proxy that handles the fetching of images from object storage (removing the need to keep them in the repo). More information is in https://grafana.com/docs/writers-toolkit/writing-guide/image-guidelines/. During local builds, Hugo injects some JavaScript that uses a websocket connection to the Hugo server process to auto-reload pages when changes are made to the underlying files. Exposing port 3003 was a quick fix to re-enable that behavior.